config/webpack.config.base.js

```diff
@@ -105,10 +110,11 @@ module.exports = {
     })
   ],
   resolve: {
-    extensions: ['.js', '.jsx'],
+    extensions: ['.js', '.jsx', '.css'],
     alias: {
       components: `${PATHS.ABS.SOURCE}/components`,
-      config: `${PATHS.ABS.SOURCE}/config`
+      config: `${PATHS.ABS.SOURCE}/config`,
```

Comment:
well, an alias allows you to do:

`import Foo from 'config/Foo'`

as opposed to:

`import Foo from '../../config/Foo';`

so if you can do the first without problems, then yes, it works. if not, then something is not quite right. in the case of no, we should just get rid of these aliases, and I'll figure it out when I have some time.